Software/Python/easygopigo3.py

```diff
@@ -293,12 +293,41 @@ def turn_degrees(self, degrees, blocking=False):
                                 (StartPositionLeft + WheelTurnDegrees))
         self.set_motor_position(self.MOTOR_RIGHT,
                                 (StartPositionRight - WheelTurnDegrees))
-        
+
         if blocking:
             while self.target_reached(
                     StartPositionLeft + WheelTurnDegrees,
                     StartPositionRight - WheelTurnDegrees) is False:
                 time.sleep(0.1)
+
+    def create_light_sensor(self, port):
+        return LightSensor(port, self)
+
+    def create_sound_sensor(self, port):
+        return SoundSensor(port, self)
+
+    def create_ultrasonic_sensor(self, port):
+        return UltraSonicSensor(port, self)
+
+    def create_buzzer(self, port):
+        return Buzzer(port, self)
+
+    def create_led(self, port):
+        return Led(port, self)
+
+    def create_button_sensor(self, port):
+        return ButtonSensor(port, self)
+
+    def create_linefollower(self, port):
```

Comment:
Ok, after talking a bit with @TarynSullivan  we're going with assign_button_sensor etc..
Just change create for assign